Screens/thirdScreen.js

```diff
@@ -5,182 +5,21 @@ import { StyleSheet, Button, View, SafeAreaView, Text, Alert, TouchableHighlight
-  const [thirdButton, setThirdButton] = useState(false); 
-  function navigate(){
-    navigation.navigate("secondScreen")
-  }
-  function togNext(bNext){
-    if(bNext){
-      Alert.alert("You may pass");
-      //navigate()
-    }else{
-      Alert.alert("Please select a response");
-    }
-  }
-  function togButton(buttonNumber){
-    //Alert.alert('Button pressed');
-    if(buttonNumber == 0 && firstButton){
-      setFirstButton(false);
-      setNextButton(false);
-    }else if(buttonNumber == 1 && secondButton){
-      setSecondButton(false)
-      setNextButton(false);
-    }else if(buttonNumber == 2 && thirdButton){
-      setThirdButton(false)
-      setNextButton(false);
-    }else if(buttonNumber == 0){
-      setFirstButton(true);
-      setSecondButton(false);
-      setThirdButton(false);
-      setNextButton(true);
-      setResponse("Yes, limited a lot");
-    }else if(buttonNumber == 1){
-      setFirstButton(false);
-      setSecondButton(true);
-      setThirdButton(false);
-      setNextButton(true);
-      setResponse("Yes, limited a little"); 
-    }else if(buttonNumber == 2){
-      setFirstButton(false);
-      setSecondButton(false);
-      setThirdButton(true);
-      setNextButton(true);
-      setResponse("No, not at all")
-    }
-  }
-  function ResponseFunc(){
-    if(firstButton || secondButton || thirdButton){
-      return(<View><Text>{response}</Text></View>);
-    }
+  const types = ["Laurel", "John"]
+  function TheRendered(){
+    return(
+      QRender(types)
+    );
   }
   return(
-    <SafeAreaView style={styles.container}>
-    
-      <View>
-        <Text style={styles.head_title}>
-          How does <Text style= {styles.title_bold}>your health limit you</Text> in 
-        </Text>
-        <Text style={styles.title}>
-          Vigorous activities, such as running, lifting heavy objects, participating in strenuous sports
-        </Text>
-      </View>
-      {/* <SafeAreaView  style={styles.container}> */}
-
-      <View style={bStyle(firstButton)}>
-        <Button
-          title= "Yes, limited a lot "
-          color= {txtColor(firstButton)}
-          onPress={() => togButton(0)}
-        />
-      </View>
-      <Separator/>
-      <View style={bStyle(secondButton)}>
-        <Button
-          title= "Yes, limited a little "
-          color= {txtColor(secondButton)}
-          onPress={() => togButton(1)}
-        />
-      </View>
-      <Separator/>
-      <View style={bStyle(thirdButton)}>
-        <Button
-          title= "No, not limited at all"
-          color={txtColor(thirdButton)}
-          onPress={() => togButton(2)}
-        />
-      </View>
-      <View style = {styles.filler}/>
-      <View style = {bStyle(nextButton)}>
-        <Button 
-          title= "Continue" 
-          color= {txtColor(nextButton)}
-          onPress= {() => togNext(nextButton)}
-        />
-      </View>
-      <ResponseFunc/>
-    </SafeAreaView>
+    <View>
+      <TheRendered/>
```

Comment:
this is not a specific name. i am nit-picking now but in the future/in a bigger codebase this would be too vague